frontend/src/components/Forum/Post.jsx

```
@@ -64,64 +67,107 @@ const Post = ({ post, onDelete }) => {
+
+    {/* Tags */}
+    <div className="flex flex-wrap gap-2 mt-4">
+      {updatedPost.tags?.map((tag) => (
+        <button
+          key={tag.id}
+          className="bg-blue-200 text-blue-800 px-3 py-1 rounded-full text-xs font-medium hover:bg-blue-300 transition"
+          onClick={() =>
+            window.dispatchEvent(new CustomEvent("tagClicked", { detail: tag.name }))
+          }
+        >
+          #{tag.name}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)